en/contributing/build.md

```diff
@@ -111,6 +111,11 @@ To build the *DroneCore* C++ Library on Windows:
      ```sh
      cmake --build . --config Release
      ```
+   <span></span>
```

Comment:
I added this span. Due to bugs in gitbook it is needed if you render two notes/tips/warnings immediately after each other.